packages/forms/src/components/CountryPicker/index.js

```diff
@@ -120,6 +104,250 @@ CountryPicker.defaultProps = {
     readOnly: false,
     required: false,
     error: undefined,
+    countries: [
```

Comment:
I think by default we should use undefined and internally when it is not an array then we always return all the countries